doc/configuration.rst

```diff
@@ -584,6 +584,52 @@ seconds, the following can be used:
 
    confluence_timeout = 10
 
+
+.. |confluence_publish_subset| replace:: ``confluence_publish_subset``
+.. _confluence_publish_subset:
+
+confluence_publish_subset
```

Comment:
Mind if I do some of my own tweaks for the proposed documentation changes (and stack them in this pull request)?

Reply:
I don't mind @jdknight 
tweak away :)